### PR TITLE
Pull Request for Adding 'Microverse template' files

### PR DESCRIPTION
**Pull request by jkanyi-web**
- Added Victor-Ahmed-Repo file
- Added Victor-Ahmed-Repo-main.zip file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```
Comment (jkanyi-web) [inline index.html:36]: Please remove unnecessary comments from your codebase as they make your code unclean

File: index.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+  <body>
+    <div id="main">
+      <title>VictorAhmed</title>
+      <div id="img-div"><img src="https://images.unsplash.com/photo-1551150441-3f3828204ef0?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=870&q=80"  alt="logo">
+      <p id="img-caption">Horse Logo in black background</p>
+      </div>
+      <p id="tribute-info">Lorem ipsum dolor, sit amet consectetur adipisicing elit. Maiores inventore distinctio nihil ex suscipit. Omnis, officiis distinctio pariatur in excepturi dignissimos! Nobis maxime accusantium neque, ipsa officiis suscipit voluptates reprehenderit, quia eos, inventore sit nam dignissimos. Quo, repudiandae? Harum perferendis reprehenderit, eveniet amet fugit saepe.</p>
+      <a href="alibaba.com" target="_blank" id="tirbute-link">Alibaba</a>
+    </div>
```
Comment (jkanyi-web) [inline index.html:18]: Please use proper html tag semantics, for example, you could use a section tag in place of that div#main tag. This is the new proper way of using html tags.
Also use proper indentation so as to make your code neat and clean and also for easy accessibility.
Please use a descriptive alt attribute for your logo image. This enables screen readers to enable users with disability understand what the logo represents.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#img-div img{
+    text-align: center;
+    height: 200px;
+     width: 200px;
```
Comment (jkanyi-web) [inline style.css:4]: Please use proper indentation for your stylesheet in order to enhance neatness and presentability of your code.
Linters are a great tool to enable you achieve this. Kindly view this page to understand linters and how to work with them to enhance your code https://github.com/microverseinc/linters-config/tree/master/html-css

**Review by jkanyi-web — COMMENTED** [review]
# Changes Required ♻️ 
Please check my comments below and make all required changes.

Submit for re-review when you're done.
Thanks 😀

**Review by Olamarx — APPROVED** [review]
Hi @jkanyi-web,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- GitHub review feature ✔️
- Inline comments ✔️ 
- Polite comments ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**